floodviz/map_utils_tests.py

```diff
@@ -0,0 +1,74 @@
+from floodviz.map_utils import site_dict, write_geojson
+
+import unittest
+
+import requests
+import requests_mock
+
+sites = ['05463500', '05471050', '05420680', '05479000', '05484000', '05481000', '05486000', '05421000', '05485500',
```

Comment:
Test data should be set up in the test class. That's what the setUp function is for. This also guarentees that your test data is the same for each test since it is constructed for each test. If you declare them here, any test could change them, affecting the next test that is run. Alternatively, set up your test data in the individual test that requires it.

Just stylistically, it's not necessary to have this many sites. Testing with a smaller number (say three or four) will make your tests less bulky.